frontend/src/css/Dashboard.styl

```diff
@@ -59,6 +59,10 @@
     flex-wrap wrap
     padding 30px
     min-height 100%
+    justify-content space-around
+    .subTitle
```

Comment:
Maybe
```suggestion
    & .subTitle
```

Reply:
It's equivalent and not necessary as far as I know

Reply:
:ok_hand: